### PR TITLE
[21] - Linear scroll for scroll down arrows;

### PR DESCRIPTION
**Pull request by kirilknysh**

### CODE REVIEW

File: public/landing/landing.coffee [hunk 1]
```diff
@@ -22,3 +23,32 @@ initHideOnScroll = (scrollLimit) ->
...
+		currentPosition += (timeStamp - frameStart) / 10 / scrollDuration * scrollLength
+		currentPosition = targetPosition if currentPosition > targetPosition
+
+		window.scrollTo 0, currentPosition
+
+		if currentPosition < targetPosition
+			window.requestAnimationFrame scrollStep
+
+	if arrows
+		arrows.addEventListener 'click', ->
```
Comment (AndreyLukashenko) [inline public/landing/landing.coffee]: Maybe you could use nxt.Event

Comment (DaQuirm) [inline public/landing/landing.coffee]: But we're not using nexus for the landing, are we?
I don't know

Comment (kirilknysh) [inline public/landing/landing.coffee]: No, no Nexus on Landing page - it's an overhead

File: public/landing/landing.coffee [hunk 2]
```diff
@@ -22,3 +23,32 @@ initHideOnScroll = (scrollLimit) ->
 			node.style.opacity = newOpacity
 
 		lastOpacity = newOpacity
+
+initScrollDownArrows = (scrollDuration) ->
+	arrows = document.getElementsByClassName('scroll-down-container')[0]
```
Comment (DaQuirm) [inline public/landing/landing.coffee]:
```coffeescript
[arrows] = document.getElementsByClassName 'scroll-down-container'
```

Comment (kirilknysh) [inline public/landing/landing.coffee]: Done;

**Comment by DaQuirm** [comment]
LGTM

**Comment by R1ZZU** [comment]
+1

**Pull request merged**